salt/metalk8s/salt/master/configured.sls

```diff
@@ -3,9 +3,9 @@
 
 {%- set ip_candidates = salt.network.ip_addrs(cidr=networks.control_plane) %}
```

Comment:
Oh, these are leftovers... this `ip_candidates`, `if ip_candidates` (and fallback to `127.0.0.1`) etc. should be no longer required. `salt_ip` should simply be the `metalk8s:control_plane_ip` *grain* of the minion (also in the other instance below).

I think it makes sense to also clean this up as part of this PR.